docs/resources/dsm_app.md

```diff
@@ -21,9 +66,51 @@ The following arguments are supported in the `dsm_app` resource block:
 
 * **name**: The Fortanix DSM App name
 * **default_group**: The Fortanix DSM group object id to be mapped to the app by default
-* _**other_group (optional)**_: The Fortanix DSM group object id the app needs to be assigned to
+* _**other_group (optional)**_: The Fortanix DSM group object id the app needs to be assigned to. If you want to 
```

Comment:
I know you didn't introduce it, but "other_group" is a _terrible_ name for this field

Reply:
Yes, I agree. It should be add_group. But if we change it, it will effect existing customers.